src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractImportCompletionProvider.cs

```diff
@@ -205,19 +204,15 @@ private async Task<bool> IsInImportsDirectiveAsync(Document document, int positi
                 && !IsFinalSemicolonOfUsingOrExtern(node, leftToken);
         }
 
-        protected static bool IsAddingImportsSupported(Document document)
+        protected static bool IsAddingImportsSupported(Workspace workspace, Document document, bool disallowAddingImports)
```

Comment:
why pass workspace and document?  former seems redundant.

Reply:
Will remove.